proto/pdpb.proto

```diff
@@ -113,6 +119,53 @@ message WatchGlobalConfigResponse {
     ResponseHeader header = 3;
 }
 
+// copied part of https://github.com/etcd-io/etcd/blob/7dfd29b0cc7ce25337276dce646ca2a65aa44b4d/api/etcdserverpb/rpc.proto
+message WatchRequest {
+    bytes key = 1;
+    bytes range_end = 2;
+    int64 start_revision = 3;
+}
+
+// copied part of https://github.com/etcd-io/etcd/blob/7dfd29b0cc7ce25337276dce646ca2a65aa44b4d/api/etcdserverpb/rpc.proto
+message WatchResponse {
+    ResponseHeader header = 1;
+    int64 compact_revision = 2;
+    int64 revision = 3;
+    repeated Event events = 4;
+}
+
+// copied part of https://github.com/etcd-io/etcd/blob/7dfd29b0cc7ce25337276dce646ca2a65aa44b4d/api/etcdserverpb/rpc.proto
+message GetRequest {
+    bytes key = 1;
+    bytes range_end = 2;
+    int64 limit = 3;
+    int64 revision = 4;
+}
+
+// copied part of https://github.com/etcd-io/etcd/blob/7dfd29b0cc7ce25337276dce646ca2a65aa44b4d/api/etcdserverpb/rpc.proto
+message GetResponse {
+    ResponseHeader header = 1;
```

Comment:
I think the response header should be a trimmed of this:
https://github.com/etcd-io/etcd/blob/3cf2e79e16a7a0e08de0ac168c829b4df20ae644/api/etcdserverpb/rpc.proto#L391-L406

(At least I think we need to keep the `revision` field of this...)

Reply:
Done. Add the revision in the corresponding response.